package.json

```diff
@@ -34,7 +34,7 @@
     "babel-eslint": ">=8.0.0",
     "eslint-config-prettier": ">=2.6.0",
     "eslint-plugin-flowtype": ">=2.36.0",
-    "eslint-plugin-graphql": ">=2.0.0",
```

Comment:
Won't `">=2.0.0"` already allow for 2.1?

Reply:
Oh lol that's a good point. I got all exicted and opened up the PR. This should just work™️ in devtools now?

Reply:
Hmm it's not updating on my machine:

```sh
☁  devtools [master] rm -rf node_modules
☁  devtools [master] npm install

> fsevents@1.1.3 install /Users/koddsson/github/devtools/node_modules/fsevents
> node install

[fsevents] Success: "/Users/koddsson/github/devtools/node_modules/fsevents/lib/binding/Release/node-v59-darwin-x64/fse.node" is installed via remote

> uglifyjs-webpack-plugin@0.4.6 postinstall /Users/koddsson/github/devtools/node_modules/uglifyjs-webpack-plugin
> node lib/post_install.js

added 1772 packages in 67.799s
☁  devtools [master] cat node_modules/eslint-plugin-graphql/package.json | jq .version
"2.0.0"
```